Logstash/logstash-input-azurewadtable/lib/logstash/inputs/azurewadtable.rb

```diff
@@ -52,14 +53,34 @@ def run(output_queue)
   def teardown
   end  
 
-  def process(output_queue)
-    @logger.debug(@last_timestamp)
+  def build_latent_query
```

Comment:
Data from a single VM may still be lost. It does not happen either because there is not enough concurrency or because RowKey generation algorithm prevents it at the moment. However Azure Table query paging realization with continuation tokens does not guarantee that new data will be returned by the old query. So, the scenario with 0 delay even with 1 VM still has bug you have discovered. And I agree there might be cases with more than minute delay.
I'd suggest 
- setting delay to 1 minute by default.
- leaving a comment with link to the issue #23 explaining why it's dangerous to set delay to 0